larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -318,21 +333,23 @@ TEST_P(BConv2DOpTest, SimpleTest) {
   const int stride_width = param.stride_width;
   const int dilation_height_factor = param.dilation_height_factor;
   const int dilation_width_factor = param.dilation_width_factor;
-  // TOOD: currently we are ignoring padding
   const Padding padding = param.padding;
   const int num_threads = param.num_threads;
 
+  const Padding tflite_padding =
+      (padding == Padding_ONE ? Padding_VALID : padding);
+
   const int input_num_elem =
       input_batch_count * input_height * input_width * input_depth;
 
   const int filters_num_elem =
       filter_height * filter_width * input_depth * filter_count;
 
   using T = float;
-  std::vector<T> input_data, filters_data;
+  std::vector<T> bconv_input_data, tflite_input_data, filters_data;
```

Comment:
by `tflite_input_data` you mean the `builtin_op_input_data`? why the builtin op and bconv should have different inputs?

Reply:
`builtin_op_input_data` is indeed a better name :+1: .

The reason for different inputs is that the tflite input needs to be padded with +1s. (see other comment above)